app/services/steam_importer_service.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require "net/http"
```

Comment:
No need for the requires, rails will do the requires for you

Reply:
Removed :)